### PR TITLE
12.0 mig html image url extractor

### PR DESCRIPTION
**Pull request by MrFangD**

### CODE REVIEW

File: html_image_url_extractor/models/ir_fields_converter.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Copyright 2016-2017 Jairo Llopis <jairo.llopis@tecnativa.com>
```
Comment (elicoidal) [inline html_image_url_extractor/models/ir_fields_converter.py:1]: not sure why this file is not rendered properly in github.

Comment (elicoidal) [reply html_image_url_extractor/models/ir_fields_converter.py:1]: update to 2018 everywhere

**Review by elicoidal — APPROVED** [review]
LGTM

**Review by yajo — APPROVED** [review]
Due to [some error in the past when migrating to v9](https://github.com/OCA/server-tools/pull/624), I just noticed the 1st commit 4a89640 is not attributed to me as [the original author](https://github.com/OCA/server-tools/pull/354).

It's not your fault, nor anything too important, so I approve the PR, but I'd personally appreciate it if you can restore it.

Thanks!

**Comment by pedrobaeza** [comment]
@Yajo on the migration to v9, the commit was correct. It has been later when the attribution was gone.

**Comment by yajo** [comment]
Well, not sure when, but [the commit in v9 branch is altered](https://github.com/OCA/server-tools/commit/cec41e8c007fdc4b5050889d4c255b01c668eacd#diff-ca0811620a6ebe79e19633d0181da54e). It seems that PR was squashed on merge.

**Comment by pedrobaeza** [comment]
Yes, it was on merge, not on the PR. @moylop260 squashed both commits on merge.

**Comment by yajo** [comment]
Closing in favor of duplicated #1508, which is more complete.

**Pull request closed without merging**